common/ssl_calls.c

```diff
@@ -866,3 +866,17 @@ ssl_tls_can_recv(struct ssl_tls *tls, int sck, int millis)
     return g_sck_can_recv(sck, millis);
 }
 
+
+/*****************************************************************************/
+const char
+*ssl_get_version(const struct ssl_st *ssl)
```

Comment:
Formatting - please move the initial "*"  to the previous line, it's a part of the return type.